webpack.login.js

```diff
@@ -0,0 +1,108 @@
+const _ = require( 'lodash' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const UglifyJSPlugin = require( 'uglifyjs-webpack-plugin' );
+const webpack = require( 'webpack' );
+
+// load in the babel config from babelrc and disable commonjs transform
+// this enables static analysis from webpack including treeshaking
+// also disable add-module-exports. TODO: remove add-module-exports from babelrc. requires fixing tests
+const babelConfig = JSON.parse( fs.readFileSync( './.babelrc', { encoding: 'utf8' } ) );
+const babelPresetEnv = _.find( babelConfig.presets, preset => preset[ 0 ] === 'env' );
+babelPresetEnv[ 1 ].modules = false;
+_.remove( babelConfig.plugins, elem => elem === 'add-module-exports' );
+
+const cacheIdentifier = require( './server/bundler/babel/babel-loader-cache-identifier' );
+
+const babelLoader = {
+	loader: 'babel-loader',
+	options: Object.assign( {}, babelConfig, {
+		babelrc: false,
+		cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
+		cacheIdentifier: cacheIdentifier,
+		plugins: [
+			...babelConfig.plugins,
+			'/Users/yury/Automattic/wp-calypso/inline-imports.js',
+		],
+	} ),
+};
+
+const config = {
+	devtool: 'source-map',
+	entry: {
+		build: path.join( path.resolve( __dirname ), 'client', 'boot', 'loggedout-app' ),
+		vendor: [
+			'classnames',
+			'create-react-class',
+			'gridicons',
+			'i18n-calypso',
+			'immutable',
+			'lodash',
+			'moment',
+			'page',
+			'prop-types',
+			'react',
+			'react-dom',
+			'react-redux',
+			'redux',
+			'redux-thunk',
+			'social-logos',
+			'store',
+			'wpcom',
+			'debug',
+		],
+	},
+	output: {
+		filename: '[name].[chunkhash].min.js', // prefer the chunkhash, which depends on the chunk, not the entire build
+		chunkFilename: '[name].[chunkhash].min.js', // ditto
+		path: path.resolve( 'login-build' ),
```

Comment:
Sometimes we talk about having a bundle only for the `Login` page, sometimes it's for logged-out users. I think it would be a good idea to clarify our focus, and be consistent in our code.